kodev

```diff
@@ -105,7 +105,10 @@ function setup_env() {
         # for android adb install
         KODEBUG_SUFFIX=-debug
     fi
-    readarray -d '' files < <(find . -maxdepth 1 -name 'koreader-emulator-*' -print0 | ${SETUP_ENV_GREP_COMMAND})
+    files=()
+    while IFS=  read -r -d $'\0'; do
+        files+=("$REPLY")
+    done < <(find . -maxdepth 1 -name 'koreader-emulator-*' -print0 | ${SETUP_ENV_GREP_COMMAND})
```

Comment:
The only thing I mean by not in the current Docker image is that any PR that requires a change to it can't be merged, and of course implicitly that as a rule of thumb I prefer not to change it but it needs the occasional maintenance anyway. ;-)

Reply:
Oh, no worries, this is exactly why I asked: I had the more verbose but more compatible code on hand, I just wanted to check if we could avoid it ;).